### ISSUE TITLE
Alarms issued from C++ not being reported

### ISSUE THREAD

**Issue by dtheobald** [Metaswitch/clearwater-snmp-handlers#36]
I changed the following code to use ipc:///var/run/clearwater/alarms instead of TCP, but C++ issued alarms are never reported:
```
clearwater-infrastructure/clearwater-infrastructure/usr/share/clearwater/bin/alarms.py:63
clearwater-snmp-handlers/alarm_req_listener.cpp:127
cpp-common/src/alarm.cpp:214
```
However, alarms issued by the Python command - issue_alarm.py - are reported correctly.

Also, there are times after restarting the system that I have to restart snmpd to get even the Python alarms to be reports. When this happens the snmp alarm_handler doesn't have the /var/run/clearwater/alarms socket open (per lsof).

**Comment by dtheobald** [comment]
BTW - The debug trace from sprout (for instance) indicate that it has issued the alarm. I just don't know where it goes from there.

Also, it works the same regardless of whether management/signalling are split.

**Comment by mirw** [comment]
@dtheobald, as discussed, you're going to look at this in the first instance.  Let me know if you have any questions.

**Comment by mirw** [comment]
Is this really fixed?  I think https://github.com/Metaswitch/clearwater-snmp-handlers/pull/40 is still pending.  Please confirm.

**Comment by rkd-msw** [comment]
@eleanor-merry has merged #40 and confirmed that alarms work, so this is now fixed.
